src/components/pages/auth/form-auth.tsx

```diff
@@ -59,6 +64,7 @@ const AuthForm: FC<FormAuthProps> = ({ className }) => {
             </FormItem>
           )}
         />
+        <OtpInput length={5} onChange={otpChange} otpValue={otp} />
```

Comment:
Hm, I thing `length={5}` may be a constant

Reply:
Well, if we'll use otp input only on the auth pages, constant is a good oprion. length prop is for ability to change otp length dynamically, but, in case of auth, it is not necessary

Reply:
I've added an `auth` file with `LENGTH_OTP` constant in here